src/test/suite/getFixturePaths.ts

```diff
@@ -0,0 +1,18 @@
+import * as path from "path";
```

Comment:
This looks fine, but you'll want to move it into the new `test/util` directory and you'll need to resolve the merge conflict because, as mentioned yesterday, I also touched the fixture enumeration code in your other PR